packages/runtime/container-runtime/src/test/containerRuntime.spec.ts

```diff
@@ -1857,5 +1858,127 @@ describe("Runtime", () => {
 				]);
 			});
 		});
+
+		describe("GetPendingState", () => {
```

Comment:
I really like this test pattern for getPendingLocalState. I think we can do something similar to simplify all our stashed op tests

Reply:
I mean, it won't be a replacement for current stashed op tests but I can see us duplicating some coverage and using it more in subsequent tests